spec/controllers/admin/attachments_controller_spec.rb

```diff
@@ -55,7 +55,7 @@ module Alchemy
 
         context 'with params[:except]' do
           it 'does not load attachments with matching content type' do
-            get :index, except: 'jpeg'
+            alchemy_get :index, except: 'jpeg'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.